src/Api/Request/AssertRefundService.php

```diff
@@ -66,24 +66,21 @@ public function __construct(
     /**
      * @param AssertRefundRequest $assertRefundRequest
      *
-     * @return AssertBody
+     * @return array |null
      * @throws \Exception
      */
     public function assertRefund(AssertRefundRequest $assertRefundRequest)
     {
         $assertApi = self::ASSERT_REFUND_API_TRANSACTION;
+
         try {
-            $response = $this->apiRequest->post(
+            return $this->apiRequest->post(
                 $assertApi,
-                [
-                    'body' => json_encode($assertRefundRequest->getAsArray()),
-                ]
+                json_encode($assertRefundRequest->getAsArray())
             );
-        } catch (Exception $e) {
+        } catch (\Exception $e) {
```

Comment:
Its better to import class then use it with \

Reply:
fixed